### DISCUSSION TITLE
Trusted

### DISCUSSION THREAD

**Discussion by Seagoddess** [TownyAdvanced/Towny#5787] [Q&A]
How do I see trusted members of my town that are NOT residents? /plot perm screen is NOT a command and doing /plot perm add will take forever on a massive server.

**Comment by LlmDl** [comment]
`/town trust list` will show you who is trusted in your town.

**Reply by Seagoddess** [reply]
No? I tried /town trust list and also /towny trust list and even /t trust list. 

![unknown](https://user-images.githubusercontent.com/100750139/156454342-8b4fc12c-389c-470c-8694-b4cea43f1150.png)

**Reply by LlmDl** [reply]
What's your Towny version?

**Reply by Seagoddess** [reply]
I don't know

**Reply by Seagoddess** [reply]
I should mention this is not my server

**Reply by LlmDl** [reply]
Tell your admin to update their towny.
